app/HutchAgent/Services/FiveSafesCrateService.cs

```diff
@@ -377,6 +377,30 @@ public void AddLicense(ROCrate crate)
     crate.RootDataset.SetProperty("license", new Part { Id = licenseEntity.Id });
   }
 
+  public bool WorkflowIsRelativePath(ROCrate roCrate, WorkflowJob workflowJob)
```

Comment:
if you're only passing the `WorkflowJob` to get the `.WorkingDirectory.JobCrateRoot()`, why not pass that as the `string` to the method, rather than the whole object?

Also don't forget the docstring on this `pubic` method